config/locales/client.en.yml

```diff
@@ -281,3 +281,12 @@ en:
               label: URL
             channel:
               label: Channel
+
+        send_chat_integration_message:
+            title: Send Chat-Integration message
```

Comment:
is that valid indentation here? Maybe it's github tripping

Reply:
<img width="1171" alt="Screenshot 2024-09-10 at 11 24 50" src="https://github.com/user-attachments/assets/d1605d64-d699-43b2-882a-0e249defa1b6">
It took me a print to find that it was actually with weird but valid indentation